km51/Beshta_Vladyslav/laboratory1.sql

```diff
@@ -1,2 +1,52 @@
 -- LABORATORY WORK 1
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+змінювати структуру таблиць.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Beshta IDENTIFIED BY beshta
+DEFAULT TABLESPACE "USERS"
+TEMPORARY TABLESPACE "TEMP";
+
+ALTER USER Beshta QUOTA 100M ON USER;
+
+GRANT "CONNECT" TO BESHTA;
+
+GRANT ALTER ANY TABLE TO Beshta
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Книжка складається з сторінок, на яких є рядки тексту.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+CREATE TABLE BOOK(
+book_name VARCHAR(30) NOT NULL
```

Comment:
The name of the book can not be the key. The title of the books can be the same. 
You must use the book_id field for primary key.